uPortal-api/uPortal-api-rest/src/main/java/org/apereo/portal/rest/search/SearchRESTController.java

```diff
@@ -134,14 +135,24 @@
     @RequestMapping(method = RequestMethod.GET)
     public void search(
             @RequestParam("q") String query,
+            @RequestParam(value = "type", required = false) Set<String> types,
             HttpServletRequest request,
             HttpServletResponse response)
             throws IOException {
 
         final Map<String, List<?>> searchResults = new TreeMap<>();
+        if (logger.isDebugEnabled()) {
+            logger.debug(
+                    "Searching with q={}, type={}", query, ArrayUtils.toString(types.toArray()));
```

Comment:
At least some of this logging code is unnecessary.  I think the following will work nicely...

```
logger.debug("Searching with q={}, type={}", query, types);
```
Without trying it, I'm not sure if the `ArrayUtils.toString(types.toArray())` part is necessary.

Reply:
Update...
```
DEBUG [http-nio-8080-exec-2-admin] o.a.p.r.search.SearchRESTController 2019-02-26 16:18:30,738 - Searching with q=student, type=[people, portlets]
DEBUG [http-nio-8080-exec-9-admin] o.a.p.r.search.SearchRESTController 2019-02-26 16:18:44,240 - Searching with q=student, type=[portlets]
```